packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -174,10 +174,15 @@ export class LitNodeClientNodeJs
   createCapacityDelegationAuthSig = async (
     params: CapacityCreditsReq
   ): Promise<CapacityCreditsRes> => {
+    // -- validate
+    if (!params.dAppOwnerWallet) {
+      throw new Error('dAppOwnerWallet must exist');
+    }
+
     // Useful log for debugging
     if (!params.delegateeAddresses || params.delegateeAddresses.length === 0) {
       log(
-        `[createCapacityDelegationAuthSig] No delegatee addresses provided. It means that the capability will not restrict access based on delegatee list, but it may still enforce other restrictions such as usage limits (uses) and specific NFT IDs (nft_id).`
+        `[createCapacityDelegationAuthSig] 'delegateeAddresses' is an empty array. It means that no body can use it. However, if the 'delegateeAddresses' field is omitted, It means that the capability will not restrict access based on delegatee list, but it may still enforce other restrictions such as usage limits (uses) and specific NFT IDs (nft_id).`
```

Comment:
Are you sure about this? I think an empty array would mean anyone can use it?

Reply:
though i'm not sure if there's a use case where you would generate a delegation auth sig where nobody uses it lol

so @DashKash54 

```ts
const params = {
  delegateeAddresses: [] // <-- nobody can use it
};
```

```ts
const params = {
  // delegateeAddresses: [] // <-- commented out, key is omitted, meaning anyone can use it
}
```